Java/CoreNative/src/main/java/com/breadwallet/corenative/crypto/BRCryptoTransferAttribute.java

```diff
@@ -0,0 +1,52 @@
+package com.breadwallet.corenative.crypto;
+
+import com.breadwallet.corenative.CryptoLibraryDirect;
+import com.sun.jna.Pointer;
+import com.sun.jna.PointerType;
+
+public class BRCryptoTransferAttribute extends PointerType {
+    public BRCryptoTransferAttribute() {
+        super();
+    }
+
+    public BRCryptoTransferAttribute(Pointer address) {
+        super(address);
+    }
+
+    public String getKey() {
+        Pointer thisPtr = this.getPointer();
+
+        return CryptoLibraryDirect.cryptoTransferAttributeGetKey(thisPtr).getString(0, "UTF-8");
+    }
+
+    public String getValue() {
+        Pointer thisPtr = this.getPointer();
+
+        return CryptoLibraryDirect.cryptoTransferAttributeGetValue(thisPtr).getString(0, "UTF-8");
```

Comment:
Can this be null? If so, need to handle. An example would be BRCryptoPaymentProtocolRequest::getMemo.

Reply:
Fixed.  'Flowed' Optional down to the BRCryptoTransferAttribute methods.